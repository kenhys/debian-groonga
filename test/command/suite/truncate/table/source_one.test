table_create Diaries TABLE_HASH_KEY ShortText
column_create Diaries content COLUMN_SCALAR Text

table_create Terms TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram
column_create Terms content_index COLUMN_INDEX|WITH_POSITION Diaries content

dump

truncate Diaries

dump
