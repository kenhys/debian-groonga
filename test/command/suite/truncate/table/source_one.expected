table_create Diaries TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
column_create Diaries content COLUMN_SCALAR Text
[[0,0.0,0.0],true]
table_create Terms TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram
[[0,0.0,0.0],true]
column_create Terms content_index COLUMN_INDEX|WITH_POSITION Diaries content
[[0,0.0,0.0],true]
dump
table_create Diaries TABLE_HASH_KEY ShortText
column_create Diaries content COLUMN_SCALAR Text

table_create Terms TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram

column_create Terms content_index COLUMN_INDEX|WITH_POSITION Diaries content
truncate Diaries
[[0,0.0,0.0],true]
dump
table_create Diaries TABLE_HASH_KEY ShortText
column_create Diaries content COLUMN_SCALAR Text

table_create Terms TABLE_PAT_KEY ShortText --default_tokenizer TokenBigram

column_create Terms content_index COLUMN_INDEX|WITH_POSITION Diaries content
